src/views/Viewpoint/DetailViewpoint/SidebarTabsContent/DescriptionTab/index.js

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
-import { Button, Classes } from '@blueprintjs/core';
 
 import { useTranslation } from 'react-i18next';
 import CollapsiblePanel from '../../../../../components/CollapsiblePanel';
 
 import { connectViewpointProvider } from '../../../context';
-import DownloadButton from '../../../../../components/DownloadButton';
 
-const iconMapping = {
-  image: 'media',
-  xls: 'th',
+const labelFromDocument = ({ key, properties: { label } }) => label || key;
+
+const filenameFromDocument = ({ key, document, properties: { label } }) => {
```

Comment:
Maybe you can set `key` as default value of `label` to avoid  ${label || key} line 13

Reply:
done